src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -83,15 +83,18 @@
     </ItemGroup>
 
     <!--
-        Setting the property to true if patch == 0 and preview == true. SDK will set DefaultNetCorePatchVersion according to this flag.
-        So that we don't need to manually update the version selection logic between when we ship a final release and when we ship the first patch
+        Generate default runtime framework versions
       -->
-    <GetUseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion
-      BundledNETCoreAppPackageVersion="$(_NETCoreAppPackageVersion)">
-      <Output TaskParameter="UseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion"
-              PropertyName="_UseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion" />
-    </GetUseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion>
-
+    <GenerateDefaultRuntimeFrameworkVersion RuntimePackVersion="$(MicrosoftNETCoreAppRuntimePackageVersion)">
+      <Output TaskParameter="DefaultRuntimeFrameworkVersion" PropertyName="MicrosoftNETCoreAppDefaultRuntimePackageVersion" />
```

Comment:
Let's not call these "Package" versions.  Let's go with the pattern of MicrosoftNETCoreAppDefaultRuntimeFrameworkVersion, or just MicrosoftNETCoreAppDefaultRuntimeVersion